Taxi/Taxi/Presenter/MyParty/MyPartyView.swift

```diff
@@ -111,32 +110,37 @@ struct MyPartyList: View {
             } label: {
                 EmptyView()
             }
-            ScrollView {
-                LazyVStack(alignment: .leading, spacing: 10) {
-                    ForEach(meetingDates, id: \.self) { date in
-                        Section(header: MyPartySectionHeader(date: date)) {
-                            ForEach(partys[date]!, id: \.id) { party in
+            List {
+                ForEach(meetingDates, id: \.self) { date in
+                    Section(header: MyPartySectionHeader(date: date)) {
+                        ForEach(partys[date]!, id: \.id) { party in
+                            ZStack {
+                                PartyListCell(party: party)
                                 NavigationLink {
                                     ChatRoomView(party: party, user: authentication.userInfo)
                                         .environmentObject(self.viewModel)
                                 } label: {
-                                    PartyListCell(party: party)
-                                        .contentShape(Rectangle())
+                                    EmptyView()
                                 }
-                                .buttonStyle(CellButtonStyle())
-                                .disabled(isSwiped) // 스와이프 된 상태일 때 비활성화
-                                .swipeDelete(isSwiped: $isSwiped, action: {
-                                    self.showAlert = true
-                                    self.selectedParty = party
-                                })
-                                .cornerRadius(16)
-                                .cellBackground()
-                                .padding(.horizontal)
+                                .opacity(0)
                             }
+                            .disabled(isSwiped) // 스와이프 된 상태일 때 비활성화
+                            .swipeDelete(isSwiped: $isSwiped, action: {
+                                self.showAlert = true
+                                self.selectedParty = party
+                            })
+                            .contentShape(RoundedRectangle(cornerRadius: 16))
+                            .cellBackground()
                         }
                     }
+                    .listRowBackground(Color.clear)
+                    .listRowSeparator(.hidden)
```

Comment:
![image](https://user-images.githubusercontent.com/75792767/186335073-2bac4e84-ca8f-48b9-9951-7184f480123a.png)

현재 나가기 버튼이 PartyListCell보다 커서 이상한데, 나가기 버튼의 높이는 .infinity인 반면 Cell의 높이는 내부 컨텐츠 크기로 지정되어 그런것 같습니다.
List에 기본적으로 적용되는 상하 inset을 없애거나 나가기 버튼의 높이를 cell의 높이와 동일하도록 하면 될 것 같습니다.

```suggestion
                    .listRowBackground(Color.clear)
                    .listRowSeparator(.hidden)
                    .listRowInsets(EdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16))
```